relay-server/src/actors/processor.rs

```diff
@@ -2799,10 +2924,16 @@ impl EnvelopeProcessorService {
             });
         }
 
+        // Extract span metrics from standalone spans only
+        if self.inner.config.processing_enabled() || state.sampling_result.should_drop() {
+            self.extract_span_metrics(state);
+        }
+
```

Comment:
Let's call `extract_span_metrics` from within `process_spans` (see other comments).

```suggestion
```

Reply:
So, we'll only extract metrics on processing nodes?

Also, this works while we're not dynamically sampling spans, but when we do, we'll need to split this again.